XenAdmin/TabPages/ManageUpdatesPage.cs

```diff
@@ -830,6 +830,14 @@ private List<ToolStripItem> GetAlertActionItems(Alert alert)
                 items.Add(fix);
             }
 
+            if (alert is ClientUpdateAlert)
+            {
+                var download = new ToolStripMenuItem(Messages.UPDATES_DOWNLOAD_AND_INSTALL);
+                download.Click += ToolStripMenuItemDownloadInstall_Click;
+                download.Enabled = (alert as ClientUpdateAlert).Downloadable; 

```

Comment:
Double cast in lines 833 and 837. Ideally this should be written as:
`if (alert is ClientUpdateAlert cua)` and `download.Enabled = cua.Downloadable`

Reply:
This has already been merged, but we can change it in the next PR.